1-estrategia/README.md

```diff
@@ -1,23 +1,237 @@
+
+* Satisfacción del cliente
+	* Grado de satisfacción con la experiencia navegando en el sitio web (1-5) 
+
+* Grado de recomendación
+	* ¿Recomendarías este sitio web a algún amigo/familiar/conocido (poco recomendable-muy recomendable)? 
+	* NPS: Detractores (0-6), Pasivos (7-8), Promotores (9-10). 
+
+* Usabilidad
+	* ¿Qué tareas has podido completar a través de nuestro sitio web? Elabora una respuesta detallando una o varias tareas que hayas podido realizar
+
+* Dificultades del usuario
+	* ¿Con qué dificultades te has encontrado a la hora de realizar una tarea?
+
+* Descripción general del sitio
+	* ¿Con qué adjetivo/s describirías nuestro sitio web? (Escribir o seleccionar alguno de los que propongamos nosotros)
+
+* Valoración general  
+	* ¿Qué puntuación global le ofrecerías a nuestro sitio web? (1-5)
+
+## NECESIDADES DEL USUARIO
 
 ### Investigación de usuarios
+Como bien hemos comentado, nuestro proyecto web es crear una página web relacionada con la **igualdad de género**. En esta página web se encontrarán noticias, reportajes, datos, videos, etc. Todo el contenido hallado en este sitio web tendrá el siguiente objetivo: informar y educar.
+
+En cuanto a nuestro público objetivo creemos que es bastante amplio, ya que tanto mujeres como hombres mayores de 18 años podrían tener interés en acceder a nuestra página web. Pero antes de crear nuestro producto debemos conocer a nuestros usuarios y sobre todo debemos percibir al máximo sus necesidades. ¿Pero cuáles son esas necesidades? 
+Para averiguar las necesidades del usuario hemos optado por crear una **encuesta**. Mediante la encuesta conoceremos al usuario, o posible cliente, y averiguaremos sus necesidades para poder adaptarnos y ofrecerles lo que necesitan. Los cuatro conceptos claves para afrontar una encuesta y ofrecer realmente un servicio útil son: la empatía, la escucha,la observación y finalmente la obtención de conclusiones. 
+
+Para averiguar las necesidades del usuario hemos optado por crear una **encuesta**.Mediante la encuesta conoceremos al usuario, o posible cliente, y averiguaremos sus necesidades para poder adaptarnos y ofrecerles lo que necesitan. Los cuatro conceptos claves para afrontar una encuesta y ofrecer realmente un servicio útil son: la empatía, la escucha,la observación y finalmente la obtención de conclusiones. 
+
+### Características de una encuesta
+Antes de crear la encuesta hay que tener varias cosas en cuenta:
+
+•	**La encuesta tiene que tener un objetivo fijado**: nuestro objetivo es conocer las necesidades del usuario (qué quieren, cómo lo quieren, qué no quieren, etc.).
+
+•	**Intentar encuestar a aquella persona que pueda estar interesado en el servicio**: en nuestro caso encuestaremos a personas de nuestro entorno, por cercanía y accesibilidad; consideramos que nuestro entorno puede tener interés en acceder a nuestra página web, ya que nuestro público objetivo es bastante amplio: personas mayores de 18 años. 
```

Comment:
Esto es una consideración práctica que está bien poner encima de la mesa, pero de entre aquellas personas que tenéis en vuestro entorno, tendréis que identificar a aquéllas que más probablemente podrían interesarse en el tema que tratáis...